packages/openactive-broker-microservice/src/twoPhaseListeners/api.js

```diff
@@ -67,9 +67,6 @@ function doPendOrRespondToGetListenerRequest(res, listenersMap, listenerId) {
   return true;
 }
 
-/**
```

Comment:
Removed this as it broke type checking

Reply:
Is it possible to replace it with something that works and still types `req` and `res`?

Reply:
👍🏽